src/main/java/net/minecraftforge/network/NetworkInitialization.java

```diff
@@ -5,113 +5,79 @@
 
 package net.minecraftforge.network;
 
+import io.netty.util.AttributeKey;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.event.network.CustomPayloadEvent;
+import net.minecraftforge.network.packets.*;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.logging.log4j.Marker;
 import org.apache.logging.log4j.MarkerManager;
 import org.jetbrains.annotations.ApiStatus;
 
-import io.netty.util.AttributeKey;
-import net.minecraft.resources.ResourceLocation;
-import net.minecraftforge.network.packets.Acknowledge;
-import net.minecraftforge.network.packets.ChannelVersions;
-import net.minecraftforge.network.packets.LoginWrapper;
-import net.minecraftforge.network.packets.OpenContainer;
-import net.minecraftforge.network.packets.RegistryList;
-import net.minecraftforge.network.packets.RegistryData;
-import net.minecraftforge.network.packets.ConfigData;
-import net.minecraftforge.network.packets.MismatchData;
-import net.minecraftforge.network.packets.ModVersions;
-import net.minecraftforge.network.packets.SpawnEntity;
+import java.util.function.BiConsumer;
 
 @ApiStatus.Internal
 public class NetworkInitialization {
```

Comment:
```suggestion
public final class NetworkInitialization {
```
Internal class that we don't extend, so mark as final to prevent unsupported extending of internals.